benchmarks/work_precision_sets/time_vs_abstols.py

```diff
@@ -0,0 +1,126 @@
+import pybamm
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+parameters = [
+    "Marquis2019",
+    "Prada2013",
+    "Ramadass2004",
+    "Chen2020",
+]
+models = ["SPM", "DFN"]
+abstols = [
+    0.001,
+    0.0001,
+    1.0e-5,
+    1.0e-6,
+    1.0e-7,
+    1.0e-8,
+    1.0e-9,
+    1.0e-10,
+    1.0e-11,
+    1.0e-12,
+    1.0e-13,
+]
+solvers = ["IDAKLU", "Casadi - safe", "Casadi - fast"]
```

Comment:
You can do the same as with the models and define a list of PyBaMM solvers. They also have a name that you can get, but in this case comes by default and you can't set it when defining the solver. But I think it will be fine.